test/own.js

```diff
@@ -0,0 +1,574 @@
+'use strict'
+
+const test = require('node:test')
+const { EventEmitter } = require('node:events')
+const persistence = require('../')
+const { MongoClient } = require('mongodb')
+const mqemitterMongo = require('mqemitter-mongodb')
+const dbname = 'aedes-test'
+const mongourl = `mongodb://127.0.0.1/${dbname}`
+
+function sleep (msec) {
+  return new Promise(resolve => setTimeout(resolve, msec))
+}
+
+function waitForEventOnce (emitter, eventName, errorEvent) {
+  return new Promise((resolve, reject) => {
+    emitter.once(eventName, resolve)
+    if (errorEvent) {
+      emitter.once(errorEvent, reject)
+    }
+  })
+}
```

Comment:
you can replace this with `once` imported from `node:events/promises`

Reply:
Node 22.14.0 says: `Error [ERR_UNKNOWN_BUILTIN_MODULE]: No such built-in module: node:events/promises`